src/alchemlyb/convergence/convergence.py

```diff
@@ -0,0 +1,113 @@
+            10/10       3.088821       0.021573  3.089027        0.021568
+
+
+    .. versionadded:: 0.6.0
+    '''
+    logger = logging.getLogger('alchemlyb.postprocessors.'
+                               'forward_backward_convergence')
+    logger.info('Start convergence analysis.')
+    logger.info('Check data availability.')
+
+    if estimator.lower() == 'mbar':
+        logger.info('Use MBAR estimator for convergence analysis.')
+        estimator_fit = MBAR().fit
+    elif estimator.lower() == 'bar':
+        logger.info('Use BAR estimator for convergence analysis.')
+        estimator_fit = BAR().fit
+    elif estimator.lower() == 'ti':
+        logger.info('Use TI estimator for convergence analysis.')
+        estimator_fit = TI().fit
+    else:  # pragma: no cover
+        logger.warning(
+            '{} is not a valid estimator.'.format(estimator))
+
+    logger.info('Begin forward analysis')
+    forward_list = []
+    forward_error_list = []
+    for i in range(1, num + 1):
+        logger.info('Forward analysis: {:.2f}%'.format(i / num))
+        sample = []
+        for data in df_list:
+            sample.append(data[:len(data) // num * i])
+        sample = concat(sample)
+        result = estimator_fit(sample)
+        forward_list.append(result.delta_f_.iloc[0, -1])
+        if estimator.lower() == 'bar':
+            error = np.sqrt(sum(
+                [result.d_delta_f_.iloc[i, i + 1] ** 2
+                 for i in range(len(result.d_delta_f_) - 1)]))
+            forward_error_list.append(error)
+        else:
+            forward_error_list.append(result.d_delta_f_.iloc[0, -1])
+        logger.info('{:.2f} +/- {:.2f} kT'.format(forward_list[-1],
+                                                  forward_error_list[-1]))
+
+    logger.info('Begin backward analysis')
+    backward_list = []
+    backward_error_list = []
+    for i in range(1, num + 1):
+        logger.info('Backward analysis: {:.2f}%'.format(i / num))
+        sample = []
+        for data in df_list:
+            sample.append(data[-len(data) // num * i:])
+        sample = concat(sample)
+        result = estimator_fit(sample)
+        backward_list.append(result.delta_f_.iloc[0, -1])
+        if estimator.lower() == 'bar':
+            error = np.sqrt(sum(
+                [result.d_delta_f_.iloc[i, i + 1] ** 2
+                 for i in range(len(result.d_delta_f_) - 1)]))
+            backward_error_list.append(error)
+        else:
+            backward_error_list.append(result.d_delta_f_.iloc[0, -1])
+        logger.info('{:.2f} +/- {:.2f} kT'.format(backward_list[-1],
+                                                  backward_error_list[-1]))
+
+    convergence = pd.DataFrame(
+        {'Forward': forward_list,
+         'Forward_Error': forward_error_list,
+         'Backward': backward_list,
+         'Backward_Error': backward_error_list},
+        index=['{}/{}'.format(i, num) for i in range(1, num + 1)])
```

Comment:
I prefer a numerical column instead of a string. If you want to keep it as the index, also add a column `data_fraction` where you just store the float `i/num`. The numerical value is more valuable, I think, because you can directly plot against it.

Reply:
I realize my original comment might have been misleading where I talked about i/num, hope the above makes my thinking clearer.